.github/workflows/build.yml

```diff
@@ -127,6 +120,8 @@ jobs:
           apt-get -y install cargo valgrind lld git g++ clang
       - name: Checkout source code
         uses: actions/checkout@v2
+      - name: Sanity test bindings
```

Comment:
An alternative could be to also run `genbindings.sh` with the `continue-on-error` keyword. That way we could still see the output if it gets out of sync, but it doesn't prevent the job from failing.

Reply:
Hmm, I'm not sure what the value would be - if the bindings fall out of sync, `genbindings.sh` isn't going to give you a lot of useful output that `cargo check` isn't? I added it in part to short-circuit all the /stuff/ if things are stale anyway (because the intent would be one of the full-timers or regular contributors can go fix it later).

Reply:
Understanding the lifecycle for the generated code makes this clearer. I guess you would have to pair `genbindings.sh` with a `git diff` or something to show that "hey it's out of sync" since `cargo check` doesn't always identify mismatches, but that isn't really to goal so it seems fine as-is.

Reply:
The build script a bit further doesn does do some `git diff` stuff (though noting that the bindings aren't fully deterministic in terms of ordering, so letting that fluctuate a bit), but `cargo check` is a much quicker way to get that.